hypha/apply/templates/forms/includes/field.html

```diff
@@ -14,6 +14,14 @@
          {% if field.field.word_limit %}
              data-word-limit="{{ field.field.word_limit }}"
          {% endif %}
+         {% if field.field.max_length %}
+             data-maxlength="{{ field.field.max_length }}"
+             x-data="{
+                     maxlength: $el.dataset.maxlength,
+                     curlength: 12345
```

Comment:
@theskumar Any tips on best way to get the input fields current char count/length here?